crates/bevy_input/src/common_conditions.rs

```diff
@@ -115,7 +115,7 @@ mod tests {
         Schedule::default().add_systems(
             (test_system, test_system)
                 .distributive_run_if(input_toggle_active(false, KeyCode::Escape))
-                .distributive_run_if(input_pressed(KeyCode::Escape))
+                .distributive_run_if(input_pressed(&KeyCode::Escape))
```

Comment:
`input_pressed()` still takes `T` as an argument, shouldn't that signature change to `&T`? Wondering the same for `input_just_pressed` and `input_just_released`.

Reply:
yup!

Reply:
actually no, as they need the reference alive for the lifetime of the app, that makes constructing values a pain for it unless they're const or &'static